chef/cookbooks/neutron/templates/default/ovs_neutron_plugin.ini.erb

```diff
@@ -64,7 +64,7 @@
 # the server.  In most cases, the default value should be fine.
 #
 # tunnel_bridge = br-tun
-<% if @ml2_type_drivers.include?("gre") -%>
+<% if @ml2_type_drivers.include?("gre") || @ml2_type_drivers.include?("vxlan") -%>
```

Comment:
Pretty sure I remember reading about `Array#&` in about 2001.  That rich out-of-the-box API is one of the things which most attracted me to Ruby in the early days.  But I guess it's much less likely you would need `Array` intersection if you mainly do web development.